test/Basics/VerifySkipNestedDeferred.js

```diff
@@ -0,0 +1,133 @@
+﻿//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Following lines generate a baseline with flags:
+// -UseParserStateCache -ParserStateCache -ForceDeferParse -Trace:SkipNestedDeferred
+
+function assertEqual(left, right, msg) {
+    if (left != right) {

```

Comment:
I just wrote this quick assert method because I don't want the output from the UnitTestFramework to cloud up the baseline here. This non-strict equality is good enough here - we're only testing equality to primitives, I think.